lib/app/features/user/providers/report_notifier.m.dart

```diff
@@ -21,6 +21,10 @@ sealed class ReportReason with _$ReportReason {
     required String text,
     required EventReference eventReference,
   }) = ReportReasonContent;
+  const factory ReportReason.ticker({
```

Comment:
Can't we reuse the existing `ReportReasonContent`? 
Looks like the `ReportReasonTicker` and `ReportReasonContent` don't have any differences or I miss something?